pkg/app/pipedv1/plugin/azure/deployment/plugin.go

```diff
@@ -0,0 +1,78 @@
+package deployment
+
+import (
+	"context"
+	"github.com/pipe-cd/pipecd/pkg/app/pipedv1/plugin/azure/config"
+	"github.com/pipe-cd/pipecd/pkg/plugin/sdk"
+)
+
+// Plugin implements the sdk.DeploymentPlugin interface.
+type Plugin struct{}
+
+func (p *Plugin) FetchDefinedStages() []string {
+	return allStages
+}
+
+func (p *Plugin) BuildPipelineSyncStages(ctx context.Context, _ *sdk.ConfigNone, input *sdk.BuildPipelineSyncStagesInput) (*sdk.BuildPipelineSyncStagesResponse, error) {
+	return &sdk.BuildPipelineSyncStagesResponse{
+		Stages: buildPipeline(input.Request.Stages, input.Request.Rollback),
+	}, nil
+}
+
+func (p *Plugin) ExecuteStage(ctx context.Context, _ *sdk.ConfigNone, dts []*sdk.DeployTarget[config.AzureDeployTargetConfig], input *sdk.ExecuteStageInput[config.AzureApplicationSpec]) (*sdk.ExecuteStageResponse, error) {
+	switch input.Request.StageName {
+	case string(AzureFuncSync):
+		return &sdk.ExecuteStageResponse{
+			Status: p.executeAzureFuncSyncStage(ctx, dts, input),
+		}, nil
+	case string(AzureFuncSwap):
+		return &sdk.ExecuteStageResponse{
+			Status: p.executeAzureFuncSwapStage(ctx, dts, input),
+		}, nil
+	case string(AzureFuncRollback):
+		return &sdk.ExecuteStageResponse{
+			Status: p.executeAzureFuncRollbackStage(ctx, dts, input),
+		}, nil
+	default:
+		panic("unimplemented stage: " + input.Request.StageName)
+	}
+}
+
+const (
+	ArtifactAzureBlob sdk.ArtifactKind = 5
```

Comment:
```suggestion
	artifactAzureBlob sdk.ArtifactKind = 5
```

Let`s start from private